ctrd/client.go

```diff
@@ -231,9 +231,14 @@ func (c *Client) collectContainerdEvents() {
 				logrus.Warnf("failed to parse %s event: %#v", TaskExitEventTopic, out)
 				continue
 			}
-			action = "die"
+			if exitEvent.ID == exitEvent.ContainerID {
+				action = "die"
+			} else {
+				action = "exec_die"
+				attributes["execID"] = exitEvent.ID
+			}
 			containerID = exitEvent.ContainerID
-			attributes["exitcode"] = strconv.Itoa(int(exitEvent.ExitStatus))
+			attributes["exitCode"] = strconv.Itoa(int(exitEvent.ExitStatus))
```

Comment:
Why to make this exitcode uppercase? Will this bring some compatible issues for some early adopters? @HusterWan

Reply:
I will add the `event` document later